crap-advisor-frontend/src/core/components/Map/index.tsx

```diff
@@ -1,43 +1,49 @@
-import React from "react";
-import {MapContainer, TileLayer, ZoomControl} from "react-leaflet";
-import RestroomMarker from "../RestroomMarker";
-import {ApiEndpoint, defaultLocation, defaultZoom} from "../../../app/constants";
-import {Restroom} from "../../../app/typings";
-import {useDataApi} from "../../../hooks/useDataApi";
-import {toLatLngExpression} from "../../../utils";
-import "./index.css";
-import "../../../style/main.css";
-
-export interface IMapProps {
-    selectRestroom: (restroomToSelect: Restroom) => void;
+import React from 'react';
+import { MapContainer, TileLayer, ZoomControl, useMap } from 'react-leaflet';
+
+import { RestroomMarker } from 'core/components';
+
+import { defaultLocation, defaultZoom } from 'app/constants';
+import { useAppSelector } from 'app/typings/redux';
+
+import { toLatLngExpression } from 'utils';
+
+import 'style/main.css';
+
+import './index.css';
+
+function SetMapToSelectedRestroom(): null {
+  const { currentRestroom } = useAppSelector((state) => state.REVIEWS);
```

Comment:
wtf counter: 1
we use state.REVIEWS for getting RESTROOM??

Reply:
Because we pull info about current restroom in reviews reducer.

Reply:
i'll trust you with this one, but my trust is not endless...